proxy/src/coco/source.rs

```diff
@@ -217,9 +235,43 @@ pub fn blob(
         .last_commit(commit_path)?
         .map(|c| CommitHeader::from(&c));
     let (_rest, last) = p.split_last();
-    let content = match std::str::from_utf8(&file.contents) {
-        Ok(content) => BlobContent::Ascii(content.to_string()),
-        Err(_) => BlobContent::Binary,
+    let content = match (std::str::from_utf8(&file.contents), theme) {
+        (Ok(content), None) => BlobContent::Ascii(content.to_owned()),
+        (Ok(content), Some(theme)) => {
+            use syntect::easy::HighlightLines;
+            use syntect::highlighting::ThemeSet;
+            use syntect::util::LinesWithEndings;
+
+            let syntax = std::path::Path::new(path)
+                .extension()
+                .and_then(std::ffi::OsStr::to_str)
+                .and_then(|ext| SYNTAX_SET.find_syntax_by_extension(ext));
+
+            let ts = ThemeSet::load_defaults();
+            let theme = match theme {
+                Theme::Light => ts.themes.get("base16-ocean.light"),
```

Comment:
Should this be configurable?

Reply:
Perhaps eventually, but I think for now it might make sense to create our own radicle theme..